lib/bundler/audit/scanner.rb

```diff
@@ -148,9 +148,12 @@ def scan_specs(options={})
 
         @lockfile.specs.each do |gem|
           @database.check_gem(gem) do |advisory|
-            if advisory.has_supported_identifier?
-              yield UnpatchedGem.new(gem,advisory)
-            end
+            is_ignored = ignore.include?(advisory.cve_id) ||
+                         ignore.include?(advisory.osvdb_id) ||
+                         ignore.include?(advisory.ghsa_id)
+            next if is_ignored
```

Comment:
cool!  I've added an `#identifiers` method, though I would be happy to change it to `#ids` if you prefer that.  Seems to better distinguish itself since there is already a `#id` attribute.

Now the check is based on that identifiers method.  Let me know if anything looks needs changing!